crates/lib/src/install.rs

```diff
@@ -970,6 +986,22 @@ async fn install_container(
         }
     }
 
+    // For seperate /boot filesystem, the better workaround is
+    // to inject kernel arguments during installation.
+    if let Some(boot) = root_setup.boot.as_ref() {
+        if !boot.source.is_empty() {
+            let mount_extra = format!(
+                "systemd.mount-extra={}:{}:{}:{}",
```

Comment:
Let's link to  https://github.com/bootc-dev/bootc/issues/1388 here - and also note the problem is that functionality isn't in C9S today, and there were people who wanted this fix on that version.

There's more discussion in that issue, including the big wrinkle that for FIPS we need a `BOOT=` karg today.